EventListener/ViewResponseListener.php

```diff
@@ -97,11 +98,16 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
             $vars = $request->attributes->get('_template_default_vars');
         }
 
+        /** @var ViewHandlerInterface $viewHandler */
         $viewHandler = $this->container->get('fos_rest.view_handler');
 
-        if ($viewHandler->isFormatTemplating($view->getFormat())) {
+        if ($viewHandler->isFormatTemplating($view->getFormat())
```

Comment:
if we do not do this change it means that in the case of a redirect, we still set template variables, which in turn then means that instead of a redirect we trigger this code (introduced in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/451):
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/View/ViewHandler.php#L352

this issue is illustrated in the LiipHelloBundle:
https://github.com/liip/LiipHelloBundle/blob/master/Controller/ArticleController.php#L149